src/test/resources/regressions/features/termination/non-terminated_functions.gobra

```diff
@@ -0,0 +1,128 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package termination
+
+decreases x
+func f_e(x int) int {
+  if x > 2 {
+  
+    //:: ExpectedOutput(method_termination_error)
+    return f_e(f_e(x-x)+3)
+  } else {
+    return 0
+  }
+}
+
+decreases x
+func g(x int) int {
+ if x > 1 {
+   //:: ExpectedOutput(method_termination_error)
+   return h(g(x-1)-1)
+ } else {
+   return 1
+ }
+}
+
+
+//:: ExpectedOutput(postcondition_error)
+ensures result == 1
+decreases x
+func h(x int) (result int) {
+  if x > 1 {
+    return g(h(x-1)-1)
+  } else {
+    return 1
+  }
+}
```

Comment:
A comment in this example signaling that this is an example of termination checking for mutual recursion would be helpful.

Which error is reported if you omit the postcondition `ensures result == 1`?